website/src/_posts/2020-01-13-beam-2.18.0.md

```diff
@@ -30,6 +30,11 @@ See the [download page]({{ site.baseurl }}/get-started/downloads/#2180-2020-01-2
 For more information on changes in 2.18.0, check out the
 [detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12346383&projectId=12319527).
 
+Warning: If you are using Avro 1.9.x with Beam you should not migrate to this
```

Comment:
Do you think it's better to put this in the Known Issues section below?
I'm okay with moving that section up.